website/static/js/pages/view-file-tree-page.js

```diff
@@ -32,5 +32,6 @@ $(document).ready(function() {
         panelVisible.show();
         panelHidden.hide();
     });
+    $('.osf-project-navbar li#project-nav-files').addClass('active');
```

Comment:
This is a temporary workaround and should therefore be accompanied by a `TODO` or `FIXME` code comment.